demos/base-path/netlify.toml

```diff
@@ -1,7 +1,7 @@
 [build]
 command = "next build"
 publish = ".next"
-ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF ../../"
+ignore = "if [ $CACHED_COMMIT_REF == $COMMIT_REF ]; then (exit 1); else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../../plugin; fi;"
```

Comment:
Just putting a note for folks if they're not sure why we're doing an `exit 1`. See https://docs.netlify.com/configure-builds/ignore-builds/#custom-ignore-command